README.md

```diff
@@ -27,9 +27,13 @@ The initial release includes the following key features:
 
 ## Quick Start
 
-To get started with AIBrix, clone this repository and follow the setup instructions in the documentation. Our comprehensive guide will help you configure and deploy your first LLM infrastructure seamlessly.
+To get started with AIBrix, clone this repository and follow the setup instructions in the [documentation](https://aibrix.readthedocs.io/latest/getting_started/quickstart.html), which will help you configure and deploy your first LLM infrastructure seamlessly.
 
 ```shell
+# Prerequisites:
+# After installing Docker and Kubernetes, please make sure your Kubernetes cluster ready, and then run the following commands to install AIBrix components in your Kubernetes cluster.
```

Comment:
Technically, docker is not necessary, people may need other container runtimes like containerd or cri-o. `After getting Kubernetes ready` should be good. container runtime is prerequsite to run kubernetes. Most of the users will use public cloud provides kubernetes in prod, there's no need to `install` it.

Reply:
Cool updated based on review comments..